discovery-provider/src/tasks/index_rewards_manager.py

```diff
@@ -221,25 +224,31 @@ def fetch_and_parse_sol_rewards_transfer_instruction(
     """
     try:
         tx_info = solana_client_manager.get_sol_tx_info(tx_sig)
-        result: TransactionInfoResult = tx_info["result"]
+        result = tx_info.value
+        if not result:
+            raise Exception("Missing txinfo")
```

Comment:
I've seen a few different approaches to handling these cases, might wanna unify. Raising an exception seems fine to me. Just wanna make sure we log it out somewhere.

Reply:
Yep this should get logged out. The most pythonic way to handle these things is to raise and then catch with a log, so I'll update the others to match more closely

Reply:
It would get logged here https://github.com/AudiusProject/audius-protocol/blob/3209743f341a7fac1e8dbf8ee50924332028cbcf/discovery-provider/src/tasks/index_rewards_manager.py#L576